### PR TITLE
added svg hover, for styles impl

### PR DESCRIPTION
**Pull request by hicaru**
hello dear developers this is PR by ask @hecrj  separate the svg and list_picker PR.

### CODE REVIEW

**Comment by hicaru** [comment]
Please someone check this PR, very important for me

File: widget/src/svg.rs [hunk 1]
```diff
@@ -166,7 +167,11 @@ where
                 ..bounds
             };
 
-            let appearance = theme.appearance(&self.style);
+            let appearance = if is_mouse_over {
+                theme.appearance(&self.style)
+            } else {
+                theme.hovered(&self.style)
+            };
```
Comment (hecrj) [inline widget/src/svg.rs:174]: These branches need to be switched.

**Review by hecrj — CHANGES_REQUESTED** [review]

File: widget/src/svg.rs (hunk 1 above)
Comment (hicaru) [reply widget/src/svg.rs:174]: i already switched. there was ask me to separate 2 changes.

Comment (hecrj) [reply widget/src/svg.rs:174]: I mean `hovered` should be called if `is_mouse_over`, not in the `else` branch.

Comment (hecrj) [reply widget/src/svg.rs:174]:
```suggestion
            let appearance = if is_mouse_over {
                theme.hovered(&self.style)
            } else {
                theme.appearance(&self.style)
            };
```

**Pull request merged**